types.cc

```diff
@@ -2043,7 +2043,7 @@ template data_value abstract_type::deserialize_impl<>(single_fragmented_view) co
 template data_value abstract_type::deserialize_impl<>(ser::buffer_view<bytes_ostream::fragment_iterator>) const;
 template data_value abstract_type::deserialize_impl<>(managed_bytes_view) const;
 
-int32_t compare_aux(const tuple_type_impl& t, bytes_view v1, bytes_view v2) {
+int32_t compare_aux(const tuple_type_impl& t, managed_bytes_view v1, managed_bytes_view v2) {
```

Comment:
Similarily to `bytes_view`, `managed_bytes_view` is also quite a small struct, so I don't think the difference will be noticable, but it will be an improvement nonetheless, so I've changed it here and in some other places where it was applicable.